exercises/list-ops/README.md

```diff
@@ -0,0 +1,27 @@
+# List Ops
+
+Implement basic list operations.
+
+In functional languages list operations like `length`, `map`, and
+`reduce` are very common. Implement a series of basic list operations,
+without using existing functions.
+
+## Running the tests
+
+To run the tests run the command `go test` from within the exercise directory.
+
+If the test suite contains benchmarks, you can run these with the `-bench`
+flag:
+
+    go test -bench .
```

Comment:
See #1120 which was a recent change.  You can fetch and run configlet as mentioned in that PR to get this README.md updated with the "stock" changes.